### DISCUSSION TITLE
How to run web service persistently?

### DISCUSSION THREAD

**Discussion by inovve** [nicolargo/glances#2514] [General]
Installed Glances on Ubuntu 22.04, I can only access Web UI after and while running glances -w command

As soon I close SSH windows, glances get shut down. How to run persistently?

**Comment by nicolargo** [comment]
You  can use the nohup command:

    nohup glances -w

**Reply by inovve** [reply]
Didnt work
I got message "nohup: ignoring input and appending output to 'nohup.out'" and as soon I close the SSH connection service goes away

**Reply by nicolargo** [reply]
Sorry I forgot the & at the end to run the process in background:

    nohup glances -w &

But the best way to run Glances inside à service is to use systemd:

https://github.com/nicolargo/glances/wiki/Start-Glances-through-Systemd

**Reply by inovve** [reply]
thanks!
